### PR TITLE
Completed view current score feature

### LINKED ISSUE #25: US-8: As a player, I want to be able to see my current score during the game
Develop score view

### PR DESCRIPTION
**Pull request by jordie007**
US-8 feature successfully implemented

### CODE REVIEW

File: SimpCity/Game.cs [hunk 1]
```diff
@@ -223,8 +223,15 @@ public class Game {
...
-                        buildingScores[b.Info.Type].Add(b.CalcScore(calcArchive));
+                        if (b != null)
+                        {
+                            buildingScores[b.Info.Type].Add(b.CalcScore(calcArchive));
                         calcArchive.calculated(b);
+                        }
+                        else
+                        {
+                            continue;
+                        }
```
Comment (CaeNGnp) [inline SimpCity/Game.cs]: Can we have braces on the same line to keep consistent with the project formatting?
```cs
if (condition) {
...
} else {
   continue
}
```

File: SimpCity/Game.cs [hunk 2]
```diff
@@ -223,8 +223,15 @@ public class Game {
                 for (int x = 0; x < grid.Width; x++) {
                     try {
                         CityGridBuilding b = grid.Get(new CityGridPosition(x, y));
-                        buildingScores[b.Info.Type].Add(b.CalcScore(calcArchive));
+                        if (b != null)
+                        {
```
Comment (CaeNGnp) [inline SimpCity/Game.cs]: Ditto

**Review by kenwongz — APPROVED** [review]
Ok, looks good!

**Review by CaeNGnp — APPROVED** [review]
Lgtm!

**Review by JSQJ — APPROVED** [review]

**Pull request merged**